rules/linux/privilege_escalation_find_binary.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2022/02/28"
+maturity = "production"
+updated_date = "2022/02/28"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell'"
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via find Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/find/"]
+risk_score = 47
+rule_id = "6f683345-bb10-47a7-86a7-71e9c24fb358"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation","GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
```

Comment:
similar to the other rule, we can convert this to a non-sequence and add additional shells being spawned

Reply:
Also the leading wildcards can make this a costly query (*especially multiple items in an array). We can iterate out some of the common ones to account for different iterations of path such as 

```sql
and process.args in ("/bin/sh", "sh", "/usr/bin/bash", "bash"
```

Also, for linux commands and most args, we should enforce case sensitivity with `in` and `==` (instead of `:`), where possible